Package.swift

```diff
@@ -77,6 +77,10 @@ let package = Package(
             name: "NavigationExample",
             targets: ["NavigationExample"]
         ),
+        .executable(
+            name: "SplitExample",
```

Comment:
Either rename this to match `SplitNavigationApp` or the other way around (I think `SplitExample` is suitable, so renaming to `SplitApp` would be fine).